lecture3/binary-tree-set-with-iterators/modules/binary-tree-set.ts

```diff
@@ -0,0 +1,92 @@
+import type {
+  IBinaryTreeSet,
+  ITreeNode,
+  ITraversalIterator
+} from "../types/interfaces";
+import TreeNode from "./tree-node";
```

Comment:
Чтобы не слепалось визуально

```suggestion

import TreeNode from "./tree-node";
```

Reply:
Между всеми import отступ делать, или только, после/перед большой пачки импортов? Не совсем понял